libs/cri-storing-service/src/main/java/uk/gov/di/ipv/core/library/cristoringservice/CriStoringService.java

```diff
@@ -121,6 +121,7 @@ public void storeVcs(
 
             verifiableCredentialService.persistUserCredentials(vc);
             ipvSessionItem.addVcReceivedThisSession(vc);
+            ipvSessionItem.setRiskAssessmentCredential(vc.getVcString());
```

Comment:
This will set the `riskAssessmentCredential` for every VC we receive, but we only want it to happen for the TICF VC. (This method is also called in `ProcessCriCallbackHandler`)